tough/Cargo.toml

```diff
@@ -16,7 +16,7 @@ hex = "0.4.2"
 log = "0.4.8"
 olpc-cjson = { version = "0.1.0", path = "../olpc-cjson" }
 pem = "0.8.1"
-reqwest = { version = "0.11", optional = true, default-features = false, features = ["blocking"] }
+reqwest = { version = "0.11.1", optional = true, default-features = false, features = ["blocking"] }
```

Comment:
Nice!. I realized ""0.11.1" is equivalent to "^0.11.1" and which is  ">0.11.1 and < 0.12.0"